style.css

```diff
@@ -46,23 +62,24 @@ nav a {
 	justify-content: center;
 	column-gap: 20px;
 	width: 100%;
-	height: 500px;
+	height: 900px;
 	background-image: url(images/horse_head_nebula.jpg); /* a place-holder image */ 
 	background-size:cover; /* allows to get the background in the whole section*/
 }
 
 .introduction_section h2 {
-	background-color: white;
-	color: black;
+	color: white;
+	font-size: 50px;
 	font-variant: small-caps;
-	margin-right: 50px;
+	margin-right: 300px;
 }
 
 .introduction_section p {
-	background-color: black;
+	background-color: rgba(0,0,0,0.5);
 	color: white;
-	margin-right: 50px;
-	font-size: 25px;
+	margin-right: 300px;
```

Comment:
add margin-left? 
on laptop screen size text butts against left side of page

Reply:
done!